apps/bulldozer-js/.env

```
@@ -9,6 +9,7 @@ BULLDOZER_JS_SENTRY_ENVIRONMENT=# Sentry environment label for THIS deployment (
 HEXCLAVE_BULLDOZER_JS_LMDB_PATH=# folder for the on-disk LMDB store. Defaults to ./.data/bulldozer-js-lmdb
 HEXCLAVE_BULLDOZER_JS_USE_TMP_LMDB=# set to 1 to put LMDB in a throwaway temp folder (fresh every run). Handy for tests
 HEXCLAVE_BULLDOZER_JS_LOW_LEVEL_BACKEND=# set to "in-memory" to keep all data in RAM and skip LMDB. Any other value uses LMDB
+HEXCLAVE_BULLDOZER_JS_LMDB_COMPRESSION=# set to 1 to LZ4-compress LMDB values. Sticky per path: once enabled and written, keep it on. Prefer a fresh LMDB_PATH + backfill over flipping this on an existing prod store
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEXCLAVE_BULLDOZER_JS_LMDB_COMPRESSION key should go before the HEXCLAVE_BULLDOZER_JS_LMDB_PATH key

(UnorderedKey)